src/components/ResultsMap.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+	<GmapMap
+		:center="centerMap"
+		:zoom="10"
+		map-type-id="terrain"
+		style="width: 100%; height: 50vh"
+	>
+		<GmapMarker
+			v-for="(m, index) in markers"
+			:key="index"
+			:position="m.geometry.location"
+			:clickable="true"
+			:draggable="false"
+			@click="centerMapToClickedLocation(m.name)"
+		/>
+	</GmapMap>
+</template>
+
+<script>
+export default {
+  props: ['searchResults'],
+  data() {
+    return {
+			markers: this.searchResults,
+			centerMap: {
+				lat: 40, lng: -105.0465427
```

Comment:
Later we can refactor this for the default center to be the user's current location, and dynamically update based on the average latitude and longitude of all the markers from a search.